macros/schema_tests/cardinality_equality.sql

```diff
@@ -0,0 +1,28 @@
+{% macro test_cardinality_equality(model, from, to, field) %}
+
+with table_a as (
+select
+  count(1) as num_rows,
```

Comment:
Can you put the dimensions before the aggregates here (eg. `group by 1`, also in the `table_b` cte)

Any reason to use `count(1)` instead of `count(*)` here?

Reply:
re: `count(1)` instead of `count(*)`, this is strictly habitual to keep myself out of the practice of selecting *. I'm relatively positive it makes no performance difference whatsoever. Will gladly change to `count(*)` if it better fits the dbt style guidelines.